gdal/data/nitf_spec.xml

```diff
@@ -693,6 +693,107 @@
         <field name="FI_COL" length="8" type="integer"/>
     </tre>
 
+    <!-- STDI-0002-1-v5.0 Appendix AL, Section AL.6.2.4, Table AL.6-3 -->
+    <tre name="ILLUMB" minlength="381" maxlength="99985" md_prefix="NITF_ILLUMB_" location="image">
+        <field name="NUM_BANDS" length="4" type="integer"/>
+        <field name="BAND_UNIT" length="40" type="string"/>
+        <loop counter="NUM_BANDS" md_prefix="BAND_%04d_">
+            <field name="LBOUND" length="16" type="real"/>
+            <field name="UBOUND" length="16" type="real"/>
+        </loop>
+        <field name="NUM_OTHERS" length="2" type="integer"/>
+        <loop counter="NUM_OTHERS" md_prefix="OTHER_%02d_">
+            <field name="OTHER_NAME" length="40" type="string"/>
+        </loop>
+        <field name="NUM_COMS" length="1" type="integer"/>
+        <loop counter="NUM_COMS" md_prefix="COMS_%01d_">
+            <field name="COMMENT" length="80" type="string"/>
+        </loop>
+        <field name="GEO_DATUM" length="80" type="string"/>
+        <field name="GEO_DATUM_CODE" length="4" type="string"/>
+        <field name="ELLIPSOID_NAME" length="80" type="string"/>
+        <field name="ELLIPSOID_CODE" length="3" type="string"/>
+        <field name="VERTICAL_DATUM_REF" length="80" type="string"/>
+        <field name="VERTICAL_REF_CODE" length="4" type="string"/>
```

Comment:
This field mismatched with the reference metadata, but the reference MD was out of date.  Just testing line-based comments.